src/lib/db/feature-strategy-store.ts

```diff
@@ -62,21 +62,22 @@ function ensureStringValues(data: object): { [key: string]: string } {
     return Object.fromEntries(stringEntries);
 }
 
-function mapRow(row: IFeatureStrategiesTable): IFeatureStrategy {
+function mapRow(row: IFeatureStrategiesTable): FeatureStrategySchema {
     return {
         id: row.id,
+        name: 'Some-Name',
```

Comment:
I'm not sure what we should do here. If the response worked before, we shouldn't have to add a name field here now. I don't think we should use `FeatureStrategySchema` as the return type here if it makes things tricky. 

That is, we don't have to use the `*Schema` types outside our handlers. Seems good to have separate type for the response data and whatever we need internally.

Also, `name` should probably be the same as `row.strategy_name` if it stays.

Reply:
Yeah I agree here, the db layer may need to do things that the schema doesn't respect, mapping at a higher layer seems like a sane approach

Reply:
agreed